csp_nonce/utils.py

```diff
@@ -1,13 +1,14 @@
 from __future__ import unicode_literals
 import base64
-import nacl.secret
-import nacl.utils
+import os
 
+# Number of random bytes in the nonce:
+NONCE_BYTES = 24
 
 def generate_nonce():
-    """ Return a unique base64 encoded nonce hash """
-    nonce = nacl.utils.random(nacl.secret.SecretBox.NONCE_SIZE)
-    return "{}".format(base64.b64encode(nonce))
+    """ Return a unique base64 encoded nonce hash."""
+    bits = os.urandom(NONCE_BYTES)
+    return base64.b64encode(bits)
```

Comment:
If the answer is Python 3, we might want to keep digging:

Python 2:
    >>> "{}".format(base64.b64encode(os.urandom(3)))
    '7Fr+'

Python 3:
    >>> "{}".format(base64.b64encode(os.urandom(3)))
    "b'9Uq9'"

Notice it's outputing the repr value.  I think what we're looking for is actually .decode()

Python 2:
    >>> base64.b64encode(os.urandom(3)).decode()
    u'rVvX'

Python 3:
    >>> base64.b64encode(os.urandom(3)).decode()
    'gQT6'